LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Views/TipViews/Model/VPNTipsModel.swift

```diff
@@ -135,6 +139,44 @@ public final class VPNTipsModel: ObservableObject {
 
     var geoswitchingStatusUpdateTask: Task<Void, Never>?
 
+    @available(macOS 14.0, *)
+    var canShowDomainExclusionsTip: Bool {
```

Comment:
Modularizing this logic since it became more complicated than before.

The gist of these changes is we need to understand if the domain exclusion tip needs to be shown, and skip it entirely if not.